service/collector/app.go

```diff
@@ -188,11 +189,22 @@ func (a *App) getTeam(ctx context.Context, app v1alpha1.App) (string, error) {
 
 	name := key.AppCatalogEntryName(key.CatalogName(app), key.AppName(app), key.Version(app))
 
-	ace, err := a.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries(metav1.NamespaceDefault).Get(ctx, name, metav1.GetOptions{})
-	if apierrors.IsNotFound(err) {
+	var ace *v1alpha1.AppCatalogEntry
+	{
+		namespaces := []string{metav1.NamespaceDefault, "giantswarm"}
```

Comment:
```suggestion
		// Check giantswarm namespace first as it has more CRs.
		namespaces := []string{"giantswarm", metav1.NamespaceDefault}
```

Looks good but what about switching the namespaces? As it will reduce the number of API calls.

Reply:
You are right, there are more ACEs in `giantswarm` namespace than default ns.